cleansio/censor/censor.py

```diff
@@ -23,7 +24,7 @@ def __mute_explicits(self, file_path, audio_segment, timestamps):
         # Go through each word, if its an explicit, mute the duration
         muted = False
         for stamp in timestamps:
-            if stamp['word'].lower() in self.explicits: # Explicit found, mute
+            if find(sorted(self.explicits),stamp['word'].lower()): # Explicit found, mute
```

Comment:
You shouldn't be sorting every iteration of the loop. Just once at the beginning which you already did.